vault/resource_kv_secret_v2.go

```diff
@@ -364,3 +366,25 @@ func kvSecretV2Delete(_ context.Context, d *schema.ResourceData, meta interface{
 
 	return nil
 }
+
+func getKVV2SecretNameFromPath(path string) (string, error) {
```

Comment:
Would it be useful to add a table test for these functions?

Reply:
added table tests in latest commit; good call!